src/middleware/cookies.rs

```diff
@@ -38,7 +43,7 @@ impl<Data: Send + Sync + 'static> Middleware<Data> for CookiesMiddleware {
                 if let Ok(val) = hv {
                     headers.append(http::header::SET_COOKIE, val);
                 } else {
-                    //TODO Log error here
+                    // It would be useful to log this error here.
```

Comment:
Could you keep the TODO here? Those can be super useful to search for in the codebase.

Reply:
Done. Thanks